packages/ecom-hooks/README.md

```diff
@@ -0,0 +1,7 @@
+# ECOM Hooks
+
```

Comment:
Nit: I think it is clearer to name it ecom-react-hooks, since hooks are very generic, and our hooks in this package only work with react